datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mappers/DownstreamMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.linkedin.datahub.graphql.types.mappers;
```

Comment:
why is there a `DownstreamMapper` and `DoewnstreamLineageMapper`?

Reply:
Is there a better name for one of these?

Reply:
DownstreamLineage is the parent object that we get back from downstream GMS. It contains a list of "downstreams" each. This maps each one of those objects to a more generic "RelatedDataset" object. However, we are sort of at the mercy of the downstream GMS models when doing this mapping..

Reply:
Resolving